manifests/KineticWeaponry.yaml

```diff
@@ -0,0 +1,13 @@
+name: Kinetic Weaponry
+authors: Bereskatuket (Kestrel1110)
+homepage: https://github.com/kestrel1110/KineticWeaponry
+license: GPL-3.0-or-later
+version: '1.0'
+shortDescription: Adds current-day weaponry to Endless-Sky
+description: A plugin adding modern realistic weapons to Endless-Sky. They are tier 1 human weapons, already balanced to work as anti-capital ship weapons.
```

Comment:
"Endless-Sky" with a minus?

Reply:
that's a dash on my keyboard
idk
you want me to change it?

Reply:
i just removed it